decidim-core/lib/decidim/core.rb

```diff
@@ -304,6 +305,12 @@ def self.seed!
     # "MyPDFSignatureService"
   end
 
+  # The name of the class to translate user content.
+  #
+  config_accessor :machine_translation_service do
```

Comment:
We should add this option to the `decidim-generators` initializer template file too 😄